src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -215,11 +213,21 @@ std::string postQueryContext
     return answer;
   }
 
-  s = xmlTreat(cleanPayload, ciP, &parseData, RtQueryContextResponse, "queryContextResponse", NULL, &errorMsg);
+  //
+  // NOTE
```

Comment:
The same NOTE is include in other GET operations in which the same comment makes sense? Just to maintain "comment coherence" along code...